packages/configs/index.js

```diff
@@ -0,0 +1,24 @@
+let currentEnv = 'dev';
+
+const Configs = {
```

Comment:
Only class names should begin with capital, you can do it either lowercase or all caps